tripal/src/TripalDBX/TripalDbxSchema.php

```diff
@@ -1079,6 +1080,54 @@ public function getReferencingTables(
     return $db_dependencies[$cache_key];
   }
 
+ /**
+   * Create a new table from a Drupal table definition.
+   *
+   * This overrides the parent createTable() function because we want to
+   * support using postgresql UNIQUE NULLS NOT DISTINCT, because otherwise
+   * you can insert duplicate records if one column is NULL.
+   * This is only available for Postgresql versions 15 and above.
+   * @see https://github.com/GMOD/Chado/issues/139
+   *
+   * @param string $name
+   *   The name of the table to create.
+   * @param array $table
+   *   A Schema API table definition array.
+   *
+   * @throws \Drupal\Core\Database\SchemaObjectExistsException
+   *   If the specified table already exists.
+   * @throws \BadMethodCallException
+   *   When ::createTableSql() is not implemented in the concrete driver class.
+   */
+  public function createTable($name, $table) {
+    if ($this->tableExists($name)) {
```

Comment:
Add comments so we can see what we added vs. what is core drupal.
```suggestion

    // Get Drupal core to generate the create table statements just as the
    // parent version of this method does.
    // Note: this does not actually create the table yet but rather just
    // generates statements that we can modify.
    // @see Drupal\Core\Database\Schema::createTable().
    if ($this->tableExists($name)) {
```